dspace/src/main/docker-compose/environment.dev.js

```diff
@@ -0,0 +1,16 @@
+/*
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+module.exports = {
```

Comment:
This looks confusing to me.  This is a UI-side configuration that we're now copying into the backend (for Docker).

Reply:
This will allow you to test a change to the REST api with the current, stable version of the UI (when appropriate).